src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -845,6 +845,40 @@ server.3=zoo3:2888:3888</programlisting>
            </varlistentry>
 
            <varlistentry>
+            <term>rateLimiterImpl</term>
+            <listitem>
+              <para>(No Java system property)</para>
+
+              <para>Specifies an implementation of
+              org.apache.zookeeper.common.RateLimiter to use to limit the rate
+              at which a single client, identified by IP address, can make
+              connections to a single member of the ZooKeeper ensemble.</para>
+            </listitem>
+           </varlistentry>
+
+           <varlistentry>
+            <term>maxClientCnxnRate</term>
```

Comment:
Would it make sense to tie this configuration to the implementation that is being used, rather than specifying at the top level what configuration is needed for the rate limiter?